roles/openshift_hosted/tasks/storage/hostpath.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Get registry DeploymentConfig
+  oc_obj:
+    namespace: "{{ openshift_hosted_registry_namespace }}"
+    state: list
+    kind: dc
+    name: "{{ openshift_hosted_registry_name }}"
+  register: registry_dc
+
+- name: Wait for registry pods
+  oc_obj:
+    namespace: "{{ openshift_hosted_registry_namespace }}"
+    state: list
+    kind: pod
+    selector: "{% for label, value in registry_dc.results.results[0].spec.selector.iteritems() %}{{ label }}={{ value }}{% if not loop.last %},{% endif %}{% endfor %}"
+  register: registry_pods
+  until:
+  - "registry_pods.results.results[0]['items'] | count > 0"
+  # There must be as many matching pods with 'Ready' status True as there are expected replicas
+  - "registry_pods.results.results[0]['items'] | oo_collect(attribute='status.conditions') | oo_collect(attribute='status', filters={'type': 'Ready'}) | map('bool') | select | list | count == openshift_hosted_registry_replicas | int"
+  delay: 10
+  retries: "{{ (600 / 10) | int }}"
```

Comment:
6?

Reply:
@michaelgugino I'd agree with you here, I thought it was very odd too. Frankly, this was copy/pasted in large part from the gluster parts [here](https://github.com/openshift/openshift-ansible/blob/c42ef6a6963cd4337f858bf25187d0b94018e927/roles/openshift_hosted/tasks/storage/glusterfs.yml#L22)

I left it as-is because I thought it was so weird there must've been a good reason to do it that way ? Happy to change it :/

Reply:
Woops, my bad! That was supposed to be replaced by a timeout variable that defaults to 600. :)

Reply:
60 would be fine, I suppose.  Let's just put 60 in there, having an equation with two hard coded values is super silly :)